system/src/Grav/Common/Page/Medium.php

```diff
@@ -407,11 +417,15 @@ public function __call($method, $args)
      */
     public function image($variable = 'thumb')
     {
+        $locator = self::$grav['locator'];
+
+        $images_dir = $locator->findResource('image://');
```

Comment:
For relative path I suggest of doing 

    $foo = $locator->findResource('image://', false);

All resources get cached, so it's fast to receive the same path.